tests/install_istio.sh

```diff
@@ -32,61 +32,11 @@ if [ $(ver $istio_version) -lt $(ver 1.8.6) ]
 then
     echo "Istio version $istio_version is no longer officially supported by this version of Admiral"
     exit 1
+#install istio core with DNS proxying enabled and multicluster enabled
+elif [ "$os" == "osx-arm64"]; then
+    "./istio-$istio_version/bin/istioctl" install -f operator-apple-arm.yaml -y
 else
-    #install istio core with DNS proxying enabled and multicluster enabled
-# TODO Also add east-west gateway to this installation
-cat <<EOF > cluster1.yaml
-apiVersion: install.istio.io/v1alpha1
-kind: IstioOperator
-spec:
-  meshConfig:
-    defaultConfig:
-      proxyMetadata:
-        # Enable Istio agent to handle DNS requests for known hosts
-        # Unknown hosts will automatically be resolved using upstream dns servers in resolv.conf
-        ISTIO_META_DNS_CAPTURE: "true"
-  values:
-    pilot:
-      resources:
-        requests:
-          cpu: 20m
-          memory: 128Mi
-    global:
-      meshID: admiral1
-      multiCluster:
-        clusterName: admiral1
-      network: admiral1
-      proxy:
-        resources:
-          requests:
-            cpu: 20m
-            memory: 64Mi
-          limits:
-            cpu: 80m
-            memory: 256Mi
-  components:
-    ingressGateways:
-      - name: istio-eastwestgateway
-        label:
-          istio: eastwestgateway
-          app: istio-eastwestgateway
-        enabled: true
-        k8s:
-          env:
-            # sni-dnat adds the clusters required for AUTO_PASSTHROUGH mode
-            - name: ISTIO_META_ROUTER_MODE
-              value: "sni-dnat"
-          service:
-            ports:
-              - name: status-port
-                port: 15021
-                targetPort: 15021
-              - name: tls
-                port: 15443
-                targetPort: 15443
-EOF
-
-    "./istio-$istio_version/bin/istioctl" install -f cluster1.yaml -y
+    "./istio-$istio_version/bin/istioctl" install -f operator.yaml -y
```

Comment:
hey @vinay-g ... instead of separate operator yaml files, can we override the `hub` during `istioctl install`
, like something like this? `--set hub=ghcr.io/resf/istio`

Reply:
thanks will update.